__tests__/app.test.js

```diff
@@ -131,3 +132,55 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: successfully add a new comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This is very funny.",
+    };
+
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body: { comment } }) => {
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            author: "butter_bridge",
+            body: "This is very funny.",
+            article_id: 3,
+          })
+        );
+      });
+  });
+  test("400: Responds with an error message if the article_id is invalid", () => {
```

Comment:
This is a 404 because the article_id is still a number, it just doesn't exist. Please modify this test and add another test to check for an invalid id (a string).